=0.12.

```diff
@@ -0,0 +1,65 @@
+Collecting esbonio
+  Downloading esbonio-0.16.5-py3-none-any.whl.metadata (4.7 kB)
+Collecting sphinx (from esbonio)
+  Downloading sphinx-8.1.3-py3-none-any.whl.metadata (6.4 kB)
+Requirement already satisfied: platformdirs in /home/clony/v/lib/python3.12/site-packages (from esbonio) (4.3.6)
+Collecting pygls<2,>=1.1.0 (from esbonio)
+  Downloading pygls-1.3.1-py3-none-any.whl.metadata (4.7 kB)
```

Comment:
@clonder was this file added by mistake?

Reply:
Yes